scripts/dts/extract/flash.py

```diff
@@ -85,6 +85,9 @@ def _extract_partition(self, node_address):
 
         insert_defs("DT_FLASH_AREA", prop_def, prop_alias)
 
+    def _create_legacy_label(self, prop_alias, label):
+        prop_alias[label[3:]] = label
```

Comment:
Maybe `label.lstrip('DT_')` would be clearer?

Reply:
Thanks, made the change.